File: src/main/java/upbrella/be/rent/dto/response/ConditionReportPageResponse.java
package upbrella.be.rent.dto.response;

import lombok.Builder;
import lombok.Getter;

import java.util.List;

@Getter
@Builder
public class ConditionReportPageResponse {

    private List<ConditionReportResponse> conditionReports;

    public static ConditionReportPageResponse of(List<ConditionReportResponse> conditionReports) {

        return ConditionReportPageResponse.builder()
                .conditionReports(conditionReports)
                .build();
    }
}

File: src/main/kotlin/upbrella/be/rent/dto/response/ConditionReportPageResponse.kt
package upbrella.be.rent.dto.response

data class ConditionReportPageResponse(
    val conditionReports: List<ConditionReportResponse>
) {
    companion object {
        fun of(conditionReports: List<ConditionReportResponse>): ConditionReportPageResponse {
            return ConditionReportPageResponse(conditionReports)
        }
    }
}
